app/js/settings.js

```diff
@@ -120,6 +120,9 @@ const listLike = [
     "searchwords",
     "startuppages"
 ]
+const canAcceptMultipleOptions = new Set([
```

Comment:
I would propose the undo the changes to this file, and reimplement them similar to other like-like settings. You can follow the example of `favoritepages` and simply check each element in the list to be a valid language or "system", instead of implementing this new system just for this setting.

Reply:
Thanks for pointing out the place where I should start to look. It indeed seemed for me like an unnecessary big change for this kind of feature